browser-test/src/applicant/questions/file.test.ts

```diff
@@ -393,30 +399,29 @@ test.describe('file upload applicant flow', () => {
       applicantQuestions,
     }) => {
       await applicantQuestions.applyProgram(programName)
-      await applicantQuestions.answerFileUploadQuestion(
-        'file 1 content',
-        'file1.txt',
+      await applicantQuestions.answerFileUploadQuestionFromAssets(
+        'file-upload.png',
       )
-      await applicantQuestions.answerFileUploadQuestion(
-        'file 2 content',
-        'file2.txt',
+      await applicantQuestions.answerFileUploadQuestionFromAssets(
+        'file-upload-second.png',
       )
 
       await applicantQuestions.clickReview()
 
       await applicantQuestions.expectQuestionAnsweredOnReviewPage(
         fileUploadQuestionText,
-        'file1.txt',
+        'file-upload.png',
       )
 
       await applicantQuestions.expectQuestionAnsweredOnReviewPage(
         fileUploadQuestionText,
-        'file2.txt',
+        'file-upload-second.png',
       )
       await validateScreenshot(page.locator('main'), 'file-uploaded-review')
     })
 
-    test('can download file content', async ({applicantQuestions}) => {
+    // TODO remove ".fixme" once https://github.com/civiform/civiform/issues/8143 is fixed
+    test.fixme('can download file content', async ({applicantQuestions}) => {
```

Comment:
**praise:** Thanks for keeping this test here with the `.fixme` so that we can use it once the bug is resolved.  I didn't know about `.fixme`.  That's helpful!